### DISCUSSION TITLE
WatchConnectivity

### DISCUSSION THREAD

**Discussion by rastaman111** [TortugaPower/BookPlayer#783] [Q&A]
Hey!  
How do you synchronize the list of audio files between the device and the watch.  

I can’t figure out how to transfer all the files from the device to the watch that are in the FileManager

**Comment by GianniCarlo** [comment]
Hi @rastaman111 ! it should sync automatically after you play one book on your phone. The list we show in the watch is the recent books list. We still don't support stand alone playback for the watch, but we are working on cross device sync on branch v5.0.0, that will bring us one step closer to that 👍

**Reply by rastaman111** [reply]
And in what format do you store them on the watch?  

I just ran into this problem and don't understand how to implement this feature.

**Reply by GianniCarlo** [reply]
There are no files on the watch, they are just in memory objects synced from the phone, for the purpose of remote actions only

**Reply by rastaman111** [reply]
I understand correctly, we store the structure in CoreData, and display their names in the clock?
